addon/components/o-s-s/attribute-text.ts

```diff
@@ -0,0 +1,32 @@
+import { assert } from '@ember/debug';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+interface OSSAttributeTextArgs {
+  label: string;
+  value?: string;
+  copyable?: boolean;
+  tooltip?: string;
+}
+
+export default class OSSAttributeText extends Component<OSSAttributeTextArgs> {
+  @tracked hasError: boolean = false;
+  @tracked displayInitials: boolean = false;
+
+  constructor(owner: unknown, args: OSSAttributeTextArgs) {
+    super(owner, args);
+  }
+
+  get label(): string {
+    assert(`[component][OSS::Attribute::Text] label is required `, this.args.label);
+    return this.args.label;
+  }
+
+  get value(): string | undefined {
+    return this.args.value === undefined || this.args.value.length === 0 ? '-' : this.args.value;
+  }
```

Comment:
```suggestion
  get value(): string {
    return isBlank(this.args.value) ? '-' : this.args.value;
  }
```
Just check of the value is blank, otherwise return dash

Reply:
Here I need to check for undefined anyways because this.args.value is possibly undefined. but I reused the isBlank method for sure. 
Same for the copyable getter, had to juggle with the fact that it might be undefined it's a mix of what you suggested & avoiding typecheck errors